tools/sklearn/main_macros.xml

```diff
@@ -446,6 +447,10 @@ def get_cv(literal):
     <param argument="fit_intercept" type="boolean" optional="true" truevalue="booltrue" falsevalue="boolfalse" checked="@CHECKED@" label="Estimate the intercept" help="If false, the data is assumed to be already centered."/>
   </xml>
 
+  <xml name="n_jobs" token_default_value="1" token_label="The number of jobs to run in parallel">
+    <param argument="n_jobs" type="integer" min="-1" max="32" value="@DEFAULT_VALUE@" optional="true" label="@LABEL@" help="To use GALAXY_SLOTS, set to 0. Hint: n_jobs > 1 might cause job abortion with no error message."/>
```

Comment:
I would not offer this to the user.

Reply:
How about either GALAXY_SLOTS or 1? Users can choose

Reply:
The user has no change to know how many cores are available. I would not expose this.

Reply:
Okay. I'll remove the change.